src/main/webapp/dosave.jsp

```diff
@@ -461,4 +462,128 @@
 <%
         }
     } // ENDE if (speichern)
+
+if (form.equals("einzelbeleg")) {
+
+try{
+
+boolean hasZusatzNamenKommentar = EinzelbelegDB.hasZusatzNamenKommentar(String.valueOf(id));
+boolean hasLemma = EinzelbelegDB.hasLemmar(String.valueOf(id));
+
+Einzelbeleg lastEinzelbeleg = EinzelbelegDB.getById(id);
+belegform = lastEinzelbeleg.getBelegform(); // z.B. Sebastianus
+
+
+if (!hasZusatzNamenKommentar) { %>
+
+    <script>
+        $(function () {
+            var einzelbelegID = <%= id %>;
+
+            let ajaxUrl = '<%= Utils.getBaseUrl(request) %>/ajax';
+
+            // GET-Anfrage zum DetecZusatzNamenKommentar
+            $.ajax({
+                type: "GET",
+                url: ajaxUrl,
+                data: { action: "detecZusatzNamenKommentar", EinzelbelegID: einzelbelegID },
+                dataType: "json",
+                success: function (data) {
+                    console.log(data);
```

Comment:
Please remove all console.log() statements that you added for debug purposes

Reply:
done